### PR TITLE
Fixes #199 by making axs.constants.GLOBAL_PROPERTIES the set

### LINKED ISSUE #199: UnsupportedAriaRoles has false positives on global attributes
e.g. `<div aria-hidden="true">` is returned as an error.

### PR DESCRIPTION
**Pull request by alice**
...rather than the list.

@RickSBrown PTAL

### CODE REVIEW

File: src/js/Properties.js [hunk 1]
```diff
@@ -703,8 +703,7 @@ axs.properties.getAriaProperties = function(element) {
  */
 axs.properties.getGlobalAriaProperties = function(element) {
     var globalProperties = {};
-    for (var i = 0; i < axs.constants.GLOBAL_PROPERTIES.length; i++) {
-        var property = axs.constants.GLOBAL_PROPERTIES[i];
+    for (var property in axs.constants.GLOBAL_PROPERTIES) {
```
Comment (RickSBrown) [inline src/js/Properties.js]: i'd consider adding ` && axs.constants.GLOBAL_PROPERTIES.hasOwnProperty(property)` but it's not truly necessary here

**Comment by RickSBrown** [comment]
@alice LGTM
This is great, i totally missed `propertiesSet` in PR #195, * facepalm * - sorry.

I think this bug is a duplicate of #178 so this PR should fix both and we can just close #195.

**Comment by alice** [comment]
Oh man I'm so :flushed: that I forgot about your PR and the other bug! I'll dup my bug to #178 - do you want to pull my changes into your PR?

**Comment by RickSBrown** [comment]
Ok I'll incorporate your changes into my PR.

**Pull request merged**